doc/source/ray-air/examples/dreambooth_finetuning.rst

```diff
@@ -23,7 +23,6 @@ This example fine-tunes both the ``text_encoder`` and ``unet`` models used in th
 
 
 .. image:: images/dreambooth_example.png
-   :target: images/dreambooth_example.png
    :alt: DreamBooth example
 
 The full code repository can be found here: `https://github.com/ray-project/ray/blob/master/python/ray/air/examples/dreambooth/ <https://github.com/ray-project/ray/blob/master/python/ray/air/examples/dreambooth/>`_
```

Comment:
I think this will need to point to the new template directory?

Reply:
The images are under this `doc/source/ray-air/examples` folder though, not under the template directory. Very weird 😨

Reply:
I'll try an absolute path from source dir

Reply:
Oh oops nevermind, the images were in the template directory.